sormas-ui/src/main/java/de/symeda/sormas/ui/reports/aggregate/ReportDataGrid.java

```diff
@@ -99,8 +104,44 @@ private Button createDeleteButton(AggregateReportDto aggregateReport) {
 		return null;
 	}
 
-	public void reload() {
+	public boolean navigateTo(BaseCriteria criteria, boolean force) {
+
```

Comment:
please remove this method and pass the navigateTo method in the constructor of ReportDataGrid. The ReportDataGrid is initialized in ReportDataView which has a navigateTo.